ecosystem/indexer-grpc/indexer-grpc-post-processor/src/metrics.rs

```diff
@@ -35,3 +35,13 @@ pub static TASK_FAILURE_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
     )
     .unwrap()
 });
+
+// Processor last update time latency to current time in seconds.
+pub static PROCESSOR_LAST_UPDATED_TIME_LATENCY_IN_SECS: Lazy<GaugeVec> = Lazy::new(|| {
+    register_gauge_vec!(
+        "indexer_grpc_post_processor_processor_last_updated_time_latency_in_secs",
```

Comment:
nit: 2 `processor`s

Reply:
lol "indexer_grpc_post_processor" is the namespace and "processor_last_updated_time_latency_in_secs" is the name of the metric. 

I'll rename it to "indexer_processors_processor_last_updated_time_latency_in_secs"